data/input_files/tests/nonlinear_solid/serac_dynamic_solver/dyn_direct_solve.lua

```diff
@@ -33,18 +33,18 @@ nonlinear_solid = {
             max_iter    = 500,
             print_level = 1,
         },
+    },
 
-        dynamic = {
-            linear = {
-                rel_tol     = 1.0e-4,
-                abs_tol     = 1.0e-8,
-                max_iter    = 500,
-                print_level = 0,
-                solver_type = "gmres",
-                prec_type   = "JacobiSmoother",
-            },
-            timestepper = "SDIRK33",
+    mass_solver = {
```

Comment:
This is good for now, but I need to think about what an ideal input syntax for dynamic simulations would be.